### DISCUSSION TITLE
Make the design token object structure optional?

### DISCUSSION THREAD

**Discussion by s-h-a-d-o-w** [chakra-ui/panda#1353] [Ideas]
What I mean by that is allowing users to write this:
```ts
import { defineConfig } from '@pandacss/dev'
 
export default defineConfig({
  theme: {
    // 👇🏻 Define your tokens here
    tokens: {
      colors: {
        primary: '#0FEE0F',
        secondary: '#EE0F0F'
      },
      fonts: {
        body: 'system-ui, sans-serif'
      }
    }
  }
})
```

Because unless missing something, if you don't also specify a description, it's just unnecessary boilerplate. And based on past observations, there are many people out there who dislike that even more than me. 😅 

And it seems to me like supporting this should be straightforward. Again, unless I'm missing something, since only just recently stumbled across this project and skimmed the docs.

**Comment by astahmer** [comment]
> Design tokens in Panda are largely influenced by the [W3C Token Format(opens in a new tab)](https://tr.designtokens.org/format/).

I understand what you propose can be more convenient so here is a way to do it your way and keep the current panda behaviour as well:

```ts
/**
 * Recursively wraps each value in a { value: xxx } object
 */
export const wrapValue = (obj: Record<string, any>) => {
  const newObj: Record<string, any> = {};

  for (const key in obj) {
    if (typeof obj[key] === 'object' && !Array.isArray(obj[key])) {
      newObj[key] = wrapValue(obj[key]); // Recursive call for nested objects
    } else {
      newObj[key] = { value: obj[key] };
    }
  }

  return newObj;
};

```

```ts
import { defineConfig } from '@pandacss/dev'
 
export default defineConfig({
  theme: {
    // 👇🏻 Use wrap value here
    tokens: wrapValue({
      colors: {
        primary: '#0FEE0F',
        secondary: '#EE0F0F'
      },
      fonts: {
        body: 'system-ui, sans-serif'
      }
    })
  }
})
```

**Reply by s-h-a-d-o-w** [reply]
Hm, I hope that there would be a better way than using `any`, so that props are checked.

Anyway, I don't have a use for Panda at the moment, just suggested it because I figured it might help the project gain traction.
But I guess I should really look through the previous issues in that W3C project instead. 😅

And of course thanks for trying to help regardless! Maybe somebody else will find it useful. 🙂
